pkgs/development/python-modules/symbolic/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools-rust,
+  rustPlatform,
+  rustc,
+  cargo,
+  milksnake,
+  cffi,
+}:
+
+buildPythonPackage rec {
+  pname = "symbolic";
+  version = "10.2.1";
```

Comment:
Why not the latest 12.13.4?

Reply:
doesn't work with current glitchtip version, we tested earlier today